src/main/java/org/testng/IInvocationStatus.java

```diff
@@ -1,8 +1,12 @@
 package org.testng;
 
 public interface IInvocationStatus {
```

Comment:
Could you add some javadoc to explain what is the goal of the public interface?

Reply:
Thanks for pointing this out. This wasnt meant to be a public interface. Moved it to internal package and added javadocs